src/haddock/modules/scoring/emscoring/cns/scale_inter_final.cns

```diff
@@ -20,7 +20,7 @@ igroup
         eval($nchain2 = $nchain1 )
         while ($nchain2 < $data.ncomponents) loop nloop2
             eval($nchain2 = $nchain2 + 1)
-            eval($scalfac = $kinter * $Toppar.int_$nchain1_$nchain2)
+            eval($scalfac = 1.0)
```

Comment:
won't this impact the calculation?

Reply:
Nope - we only calculate the intermolecular interactions and the weight for scoring should simply be 1. I removed the interaction matrix from the `default.cfg` file to simplify things.